### ISSUE TITLE
声音如何设置

### ISSUE THREAD

**Issue by shu0802** [sulthonuladib/Hackintosh-NUC7I5BNH#1]
我的是nuc7，声音如何设置？HDMI输出，layout-id=3时，有时有声音，有时又没有。typeC口接上设备也不能识别。
![a61480ee5a35aad788d74ddfc87a728](https://user-images.githubusercontent.com/43750373/95683808-212cc100-0c20-11eb-9794-ef9194f3cf10.jpg)
![2d9676440b0925e71d8df87cc35bb88](https://user-images.githubusercontent.com/43750373/95683812-2984fc00-0c20-11eb-88dc-c72d30898414.jpg)

**Comment by sulthonuladib** [comment]
Can u speak english @shu0802 ? Which NUC do u use ? I mean specific version. NUC7I3BEH or something else. I think u should use another SMBIOS version, ig-platform-id and other things that relate with generation of your processor. U can read at WhateverGreen and AppleALC documentations for details.

**Comment by shu0802** [comment]
After hard work, I got to this state. I used HDMI to connect to the LG display, and I had to take off the HDMI cable and plug it back in before the sound came out. Otherwise, there will be built-in input and output. Is this yours? Where is it not set?
![2bcc242caa38b39fd930148f1277587](https://user-images.githubusercontent.com/43750373/95756929-08341680-0cd9-11eb-890f-9b86393ebeca.jpg)
![8604dbbb8d9a800dd547b3ca063a04b](https://user-images.githubusercontent.com/43750373/95756947-0f5b2480-0cd9-11eb-960a-e452d158f9a6.jpg)
![d07adee3fd35e40e03137400616914f](https://user-images.githubusercontent.com/43750373/95756964-13874200-0cd9-11eb-910a-6e7e556ae13e.jpg)
